From 6e87921a2607a6c92749759c1507043c0a9fc240 Mon Sep 17 00:00:00 2001
From: Eric Ernst <eric.ernst@intel.com>
Date: Mon, 11 Aug 2014 14:32:59 -0700
Subject: [PATCH 1/1] REVERT-ME: iio: bmm050: remove hw config restore

Issues observed when restoring HW configuration for BMM050
on platform resume.  Revert for now - long term fix to come.

Signed-off-by: Eric Ernst <eric.ernst@intel.com>
---
 drivers/iio/magnetometer/bmm050_driver.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/iio/magnetometer/bmm050_driver.c b/drivers/iio/magnetometer/bmm050_driver.c
index 46e15ca9d3c7..2426329ee29c 100644
--- a/drivers/iio/magnetometer/bmm050_driver.c
+++ b/drivers/iio/magnetometer/bmm050_driver.c
@@ -1140,6 +1140,8 @@ static int bmm_restore_hw_cfg(struct i2c_client *client)
 		(struct bmm_client_data *)i2c_get_clientdata(client);
 	int op_mode;
 
+	return 0;
+
 	mutex_lock(&client_data->mutex_op_mode);
 	err = bmm_set_op_mode(client_data, BMM_VAL_NAME(SLEEP_MODE));
 
-- 
1.7.9.5

